### ISSUE TITLE
Unexpected key(s) in state_dict: "pixel_metrics.AUPRO.fpr_limit".

### ISSUE THREAD

**Issue by 1094558673** [openvinotoolkit/anomalib#1406]
Hello, I used pixel's AUPRO metric in the training process. When I run inference code "python tools/inference/lightning_inference. py -- -- config src/anomalib/models/efficient_ad/config.yaml --weights ...” Unexpected key(s) in state_dict: "pixel_metrics.AUPRO.fpr_limit"." Error.

I saw this in an issue a few months ago, but it doesn't seem to have been fixed in the master branch.

**Comment by ashwinvaidya17** [comment]
Our inference entrypoints will change a bit with Anomalib 1.0 version. Addressing this is also a part of the refactor.

**Comment by bluehyena** [comment]
I've gone through several training sessions, but I'm facing challenges with testing on this particular issue. Could you please advise on how to proceed?

**Comment by blaz-r** [comment]
Hello @bluehyena, check #1055 for a temporary fix.

**Comment by samet-akcay** [comment]
If this PR still up-to-date, I think we should merge this

**Comment by blaz-r** [comment]
@samet-akcay I believe this issue can be closed as it was addressed in #1055.
